os_net_config/impl_nmstate.py

```diff
@@ -248,6 +248,8 @@ def __init__(self, noop=False, root_dir=''):
         self.sriov_pf_data = {}
         self.sriov_vf_data = {}
         self.migration_enabled = False
+        self.apply_pf = False
+        self.apply_vf = False
```

Comment:
Could you add a comment describing what these two booleans are for?

The variable names are probably not great. How about:

```python
self.need_pf_config = False
self.need_vf_config = False
```

Reply:
Added the comments and renamed the variable names